### PR TITLE
fix(plugins)fixed to work for phantomjs

### PR DESCRIPTION
**Pull request by LironHazan**

### CODE REVIEW

**Comment by edenpessach** [comment]
It seems that there are many tests that appears twice.
We should only duplicate tests or check the browser inside the test when that test are running on specific browser.

File: .jshintrc [hunk 1]
```diff
@@ -30,6 +30,7 @@
     "$" : false,
     "jasmine" : false,
     "$$": false,
-    "expect" : false
+    "expect" : false,
+    "xit" : false
```
Comment (edenpessach) [inline .jshintrc]: this can lead to many problems, please don't enable xit but comment the tests that are irrelevant instead.

File: spec/definitions/plugins.spec.js [hunk 2]
```diff
@@ -2,19 +2,21 @@
...
         browser.sleep(1000);
-        logger.info('navigate to imports tab');
+        logger.info('navigate to definitions tab');
         components.layout.goToDefinitions();
     });
 
-    describe('add/remove plugin', function () {
-        it('should add plugin by file zip path', function (done) {
+    describe('add/remove plugin', function() {
+        //won't work on phantomjs
```
Comment (edenpessach) [inline spec/definitions/plugins.spec.js]: open an issue in jira and add the link here.
X repeat for all those comments.

File: spec/definitions/plugins.spec.js [hunk 3]
```diff
@@ -144,65 +135,86 @@ describe('plugins section', function () {
...
+            components.modals.modal.enterUrl('https://github.com/cloudify-cosmo/cloudify-openstack-plugin/archive/master.zip');
+            components.modals.modal.save();
+
             components.layout.goToResources();
-            expect(element(by.cssContainingText('.name span', 'plugins')).isDisplayed()).toBe(true);//folder plugins should exist
-            expect(element(by.cssContainingText('.name span', 'tested-plugin')).isPresent()).toBe(false);//folder with plugin name should not exist
+            components.resources.page.openFolder(0);
+            expect(element(by.cssContainingText('.name span', 'tested-plugin')).isDisplayed()).toBe(true);//folder with plugin name should exist
+            components.resources.page.renameItem(2, 'newName');
+            browser.sleep(300);
```
Comment (edenpessach) [inline spec/definitions/plugins.spec.js]: why is this here?

File: spec/definitions/plugins.spec.js (hunk 2 above)
Comment (LironHazan) [inline spec/definitions/plugins.spec.js]: opened : COMPOSER-366
